source/user-manual/reference/ossec-conf/task-manager.rst

```diff
@@ -0,0 +1,60 @@
+.. Copyright (C) 2020 Wazuh, Inc.
+
+.. _reference_ossec_task_manager:
+
+task-manager
+============
+
+.. topic:: XML section name
+
+	.. code-block:: xml
+
+		<task-manager>
+		</task-manager>
+
+The task manager module is responsible for managing all the tasks that run on the agents. This module is only available on the manager side, on a stand-alone manager or on a master node in a cluster configuration. This module will be enabled always and cannot be deactivated. This section does not need to be defined but needs to be defined in order to change the default values.
```

Comment:
Last sentence. `This section only needs to be defined in order to change the default values`

Reply:
Changed!